index.bs

```diff
@@ -522,6 +522,12 @@ The SVG animation case is covered by the
 
 </div>
 
+Note: Within an origin with a `javascript:` URL &mdash; that is, after the
+    browser has already navigated to a `javascript:`-URL &mdash;
+    [=relative-URL strings=] may turn into `javascript:`-URLs, because the
+    origin may be used as the [=base URL=]. These cases are not handled by
+    the Sanitizer.
```

Comment:
What exactly would the base URL be here? And what is an origin with a javascript: URL? This doesn't seem to be correct.

Reply:
Agreed. I made an different wording proposal below.

Reply:
I don't think you can end up in such a situation though? Is it really a `javascript:` URL at that point or `about:blank`? You can for the base URL, except HTML already ignores it in such cases.

Reply:
Mh. What do you mean "ignores it"?

Reply:
So, after reading specs, it indeed seems that `javascript:`-Urls don't ever become base URLs. I toyed around a bit with this in the browser, and the document created by `javascript:`-navigation uses the previous document's base URL.

Spec-wise, this corresponds to:
* https://html.spec.whatwg.org/multipage/browsing-the-web.html#evaluate-a-javascript:-url, step 11, `URL` parameter,
* ditto, step 16, `about base URL` parameter,
* And https://html.spec.whatwg.org/multipage/urls-and-fetching.html#document-base-url, which references those two values.

So indeed the situation we're warning about here simply doesn't exist in a conforming browser.

I suggest we drop the PR; but I'll wait until the next meeting in case there're any objections.